### ISSUE TITLE
training and testing

### ISSUE THREAD

**Issue by ak9250** [Cluoyao/Learning-Robust-3D-Face-Reconstruction-and-Discriminative-Identity-Representation#1]
when is plan to release training and testing code with pretrained models

**Comment by Cluoyao** [comment]
The related code and pretrained models will be released next week.

**Comment by ak9250** [comment]
@Cluoyao cool looking forward to testing it out also, does this handle texture?

**Comment by Cluoyao** [comment]
No, we only considered shape information, we will consider this in the future.
